src/main/scala/com/lightbend/kafkalagexporter/ConsumerGroupCollector.scala

```diff
@@ -229,12 +229,18 @@ object ConsumerGroupCollector {
         GroupPartitionLag(gtp, offsetLag, timeLag)
       }
 
-      for((group, values) <- groupLag.groupBy(_.gtp.id)) {
-        val maxOffsetLag = values.maxBy(_.offsetLag)
-        val maxTimeLag = values.maxBy(_.timeLag)
+      for((group, groupValues) <- groupLag.groupBy(_.gtp.id)) {
+        val maxOffsetLag = groupValues.maxBy(_.offsetLag)
+        val maxTimeLag = groupValues.maxBy(_.timeLag)
 
         reporter ! Metrics.GroupValueMessage(Metrics.MaxGroupOffsetLagMetric, config.cluster.name, group, maxOffsetLag.offsetLag)
         reporter ! Metrics.GroupValueMessage(Metrics.MaxGroupTimeLagMetric, config.cluster.name, group, maxTimeLag.timeLag)
+
```

Comment:
A group level message could be created here (follow up from comment https://github.com/lightbend/kafka-lag-exporter/pull/93#issuecomment-544634558)

```suggestion
val sumOffsetLag = groupValues.sumBy(_.offsetLag)
reporter ! Metrics.GroupValueMessage(Metrics.TopicOffsetLagMetric, config.cluster.name, group, topic, sumOffsetLag)
```

Reply:
I have integrated this suggestion and included it in tests.